astroquery/esasky/core.py

```diff
@@ -0,0 +1,624 @@
+            for mission in sanatized_missions:
+                #INTEGRAL does not have a product url yet.
+                if(query_mission.lower() == self.__INTEGRAL_STRING):
+                    break
+                if(query_mission.lower() == mission.lower()):
+                    maps[query_mission] = self._get_maps_for_mission(sanatized_query_table_list[query_mission], query_mission, download_folder, cache)
+                    break
+        if(len(sanatized_query_table_list) > 0):
+            print("Maps available at %s" %os.path.abspath(download_folder))
+        else:
+            print("No maps found")
+        return commons.TableList(maps)
+    
+    def get_images(self, position, radius = 0, missions = __ALL_STRING, download_folder = __MAPS_STRING, cache = True):
+        """
+        This method gets the fits files available for the selected position and mission
+        and downloads all maps to the folder /Maps.
+        The method returns a dictionary which is divided by mission and observation id. 
+
+        Parameters
+        ----------
+        position : str or `astropy.coordinates` object
+            Can either be a string of the location, eg 'M51', or the coordinates of the object.
+        radius : str or `~astropy.units.Quantity`, optional
+            The radius of a region. Defaults to 0.
+        missions : string or list, optional
+            Can be either a specific mission or a list of missions (all mission names are found in list_missions()) 
+            or 'all' to search in all missions. Defaults to 'all'. 
+        download_folder : string, optional
+            The folder where all downloaded maps should be stored. 
+            Defaults to a folder called 'Maps' in the current working directory. 
+
+        Returns
+        -------
+        maps : `dict`
+            Each mission returns dictionary where each observation id has a HDUList. 
+            The exception is Herschel as it has multiple fits files per observation id.
+            For Herschel each observation id got the used filters with their respective HDULists.
+            It is structured in a dictionary like this:
+                dict: {'XMM-EPIC': {'observationID_x': [HDUList], 'observationID_y': [HDUList], ...}, 'HST': {'observationID_z': [HDUList], ...},
+                    'HERSCHEL': {'observationID_a': {'filter_a':[HDUList], filter_b: [HDUList]}, 'observationID_b': {'filter_a':[HDUList], filter_b: [HDUList]}, ... }}
+        
+        Examples
+        --------
+        get_images("m101", "14'", "all")
+        
+        """    
+        sanatized_position = self._sanatize_input_position(position)
+        sanatized_radius = self._sanatize_input_radius(radius)
+        sanatized_missions = self._sanatize_input_mission(missions)
+        
+        maps = dict()
+        
+        map_query_result = self.query_region_maps(sanatized_position, sanatized_radius, sanatized_missions, False)
```

Comment:
it would be helpful for readability to specify these as keywords, at least for the last parameter (what is being set to `False` here?  Hard to tell without scrolling through the code)